cmd/server/cmd/server.go

```diff
@@ -308,6 +323,9 @@ func runServer(sa *serverArgs, tmplRepo template.Repository, printf, fatalf shar
 	grpcOptions = append(grpcOptions, grpc.UnaryInterceptor(grpc_middleware.ChainUnaryServer(interceptors...)))
 
 	configManager.Register(adapterMgr)
+	if sa.useIL {
+		configManager.Register(ilEval)
```

Comment:
I really wish there was a convenient way for us to get rid of this circular dependency (eval used to construct the config manager which the eval needs to listen to). Nothing springs to mind though.